src/style.css

```
@@ -1 +1,61 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+
+body {
+  background-color: #eee;
+  margin: 60px auto;
+}
+
+header {
```

```suggestion
.header {
```
Не стилізуй по тегам. створи і використовуй класи